tests/libres_tests/res/enkf/test_ecl_config.py

```diff
@@ -1,93 +1,59 @@
-#!/usr/bin/env python
-import os.path
 from datetime import datetime
 
+import pytest
 from ecl.summary import EclSum
 
 from ert._c_wrappers.enkf import ConfigKeys, EclConfig
-from ert._c_wrappers.util import UIReturn
 
 
-def test_grid(source_root):
-    grid_file = source_root / "test-data/snake_oil_field/grid/CASE.EGRID"
-    smspec_file = (
-        source_root / "test-data/snake_oil_field/refcase/SNAKE_OIL_FIELD.SMSPEC"
-    )
-    ec = EclConfig()
-    ui = ec.validateGridFile(str(grid_file))
-    assert ui
-    assert isinstance(ui, UIReturn)
-
-    ui = ec.validateGridFile("Does/Not/Exist")
-    assert not ui
-
-    assert os.path.exists(smspec_file)
-    ui = ec.validateGridFile(str(smspec_file))
-    assert not ui
-
-
-def test_datafile(source_root):
-    ec = EclConfig()
-    ui = ec.validateDataFile("DoesNotExist")
-    assert not ui
-
+def test_ecl_config_default_num_cpu_value(source_root):
     dfile = str(source_root / "test-data/eclipse/SPE1.DATA")
-    ui = ec.validateDataFile(dfile)
-    assert ui
-    ec.setDataFile(dfile)
-    assert dfile == ec.getDataFile()
+    ec = EclConfig(data_file=dfile)
+    assert dfile == ec._data_file
+    assert 1 == ec.num_cpu
```

Comment:
Use 'ec.num_cpu == 1' instead of '1 == ec.num_cpu' (Yoda-conditions)

Reply:
ah but isn't it assert expected is equal to actual? or is it always exactly the other way around? :thinking:

Reply:
anyways it is fixed now. the real truth will remain a mystery of the cosmos to me, as i seemingly cannot remember either way.